internal/serve/httphandler/assets_handler_test.go

```diff
@@ -1379,34 +1379,6 @@ func Test_AssetHandler_submitChangeTrustTransaction_makeSurePreconditionsAreSetA
 		BaseFee: txnbuild.MinBaseFee * feeMultiplierInStroops,
 	}
 
-	t.Run("makes sure a non-empty precondition is used if none is explicitly set", func(t *testing.T) {
```

Comment:
Thank you for this 🚀 🙏

Reply:
oups I actually didn't mean to remove this one. I think it kept failing. I'll fix it in develop instead.

Reply:
Added a skip test for this one and created a ticket c587f7ad1fcff4d57c7e8e2bdc18f4cb18ac700b